apps/govquests-frontend/src/domains/authentication/services/authService.ts

```diff
@@ -0,0 +1,39 @@
+import request from "graphql-request";
+import { GENERATE_SIWE_MESSAGE } from "../graphql/generateSiweMessage";
+import { SIGN_IN_WITH_ETHEREUM } from "../graphql/signInWithEthereum";
+import { SIGN_OUT } from "../graphql/signOut";
+import { CURRENT_USER } from "../graphql/currentUser";
+import { ResultOf, VariablesOf } from "gql.tada";
+
+const API_URL =
+  process.env.NEXT_PUBLIC_API_URL || "http://localhost:3000/graphql";
+
+export const generateSiweMessage = async (
+  variables: VariablesOf<typeof GENERATE_SIWE_MESSAGE>,
+) => {
+  return await request(API_URL, GENERATE_SIWE_MESSAGE, variables, {
+    credentials: "include",
+  });
+};
+
+export const signInWithEthereum = async (
+  variables: VariablesOf<typeof SIGN_IN_WITH_ETHEREUM>,
+) => {
+  return await request(API_URL, SIGN_IN_WITH_ETHEREUM, variables, {
+    credentials: "include",
```

Comment:
`#question` what exatcly is the objective of this credentials params?

Reply:
I was trying to get the cookies working, but already figured out we don't need that 😬